packages/at_onboarding_cli/lib/src/register_cli/register.dart

```diff
@@ -64,8 +63,14 @@ class Register {
         .add(ValidateOtp())
         .start();
 
-    await activate_cli
-        .wrappedMain(['-a', params['atsign']!, '-c', params['cramkey']!]);
+    await auth_cli.wrappedMain(
+      [
+        '-a',
+        params['atsign']!,
+        '-c',
+        params['cramkey']!,
+      ],
+    );
```

Comment:
as mentioned earlier, activate_cli.dart has been deprecated as it has been entirely replaced by auth_cli.dart